qml/Ping360Status.qml

```diff
@@ -0,0 +1,52 @@
+import QtQuick 2.7
```

Comment:
What is the meaning of `og` ? 
From google:
![image](https://user-images.githubusercontent.com/1215497/61400773-5d4f6580-a8a7-11e9-98eb-ee957b01c752.png)

We should avoid this kind of alias in commits, they are not clear.

Reply:
done